schemas/alpha/azure.yaml.json

```diff
@@ -1331,6 +1309,158 @@
                     }
                 }
             }
+        },
+        "sqlDbResource": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": true,
+                "uses": true,
+                "authType": {
+                    "type": "string",
+                    "title": "The authentication type",
+                    "description": "The Azure resource supports this kind of authentication for connection.",
+                    "enum": [
+                        "USER_ASSIGNED_MANAGED_IDENTITY",
+                        "PASSWORD"
+                    ]
+                },
+                "databaseName": {
+                    "type": "string",
+                    "title": "The name of Azure resource",
+                    "description": "The name of Azure resource."
```

Comment:
Not `the name of azure resource`. Please refer to this PR: https://github.com/azure-javaee/azure-dev/pull/37/files

Reply:
Both title and description should be updated.

Reply:
Updated.